tests/unittests/utils/test_coco_polygon_clamp.py

```
@@ -0,0 +1,96 @@
+    for poly in seg:
+        xs = poly[0::2]
+        ys = poly[1::2]
+        assert all(xmin <= x <= xmax for x in xs)
+        assert all(ymin <= y <= ymax for y in ys)
+
+
+def test_polyline_to_coco_segmentation_preserves_float_and_clamps_to_image():
+    from fiftyone.utils.coco import _polyline_to_coco_segmentation
+
+    class DummyPolyline:
+        def __init__(self, points):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)